debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4

```diff
@@ -1167,7 +1167,7 @@ fragment CHARSET_NAME:               ARMSCII8 | ASCII | BIG5 | BINARY | CP1250
                                      | LATIN1 | LATIN2 | LATIN5 | LATIN7
                                      | MACCE | MACROMAN | SJIS | SWE7 | TIS620
                                      | UCS2 | UJIS | UTF16 | UTF16LE | UTF32
-                                     | UTF8 | UTF8MB4;
+                                     | UTF8 | UTF8MB3 | UTF8MB4;
```

Comment:
Hum, did something get mixed up here? It's the fix from the DBZ-1349 PR?

Reply:
It is in the original comment

Reply:
@gunnarmorling Rebased to the updated master

Reply:
Gotcha. Thx.